src/vs/workbench/contrib/debug/browser/watchExpressionsView.ts

```diff
@@ -325,6 +336,31 @@ export class WatchExpressionsRenderer extends AbstractExpressionsRenderer {
 			}
 		};
 	}
+
+	protected override renderActionBar(actionBar: ActionBar, expression: IExpression, data: IExpressionTemplateData) {
+		const contextKeyService = getContextForWatchExpressionMenu(this.contextKeyService, expression);
+		const menu = this.menuService.createMenu(MenuId.DebugWatchContext, contextKeyService);
+
+		const primary: IAction[] = [];
+		const context = expression;
+		data.elementDisposable.push(createAndFillInContextMenuActions(menu, { arg: context, shouldForwardArgs: false }, { primary, secondary: [] }, 'inline'));
+
+		actionBar.clear();
+		actionBar.context = context;
+		actionBar.push(primary, { icon: true, label: false });
+	}
+}
+
+/**
+ * Gets a context key overlay that has context for the given expression.
+ */
+function getContextForWatchExpressionMenu(parentContext: IContextKeyService, expression: IExpression, additionalContext: [string, unknown][] = []) {
```

Comment:
Looks like expression and additionalContext are unused

Reply:
Right. Just removed them.